test/lint.js

```diff
@@ -11,8 +11,8 @@ const testBrowsers = require('./test-browsers');
 const filesWithErrors = new Map();
 
 const argv = yargs.alias('version','v')
-  .usage('$0 [[--] files...]', false, (yargs) => {
-    yargs.positional('files...', {
+  .usage('$0 [[--] files...]', false, yargs => {
+    return yargs.positional('files...', {
```

Comment:
Yargs >= 12 requires that the `usage(…)` function callback returns a Yargs object.